### PR TITLE
Support Russian

### PR DESCRIPTION
**Pull request by iteplenky**
This is my first language and I know how to translate correctly.

### CODE REVIEW

**Comment by fuyutsuki** [comment]
Please refer to [en_us.ini](https://github.com/fuyutsuki/Texter/blob/release/v4.0.4/resources/language/en_us.ini) and correct the old translation.
Optionally, it would be great if you could also translate the README.md and place it as `.github/readme/ru_ru.md`.

File: resources/language/ru_ru.ini [hunk 1]
```diff
@@ -1,57 +1,72 @@
...
-command.txt.add=/txt add
-command.txt.add.success=Плавающий-Текст: {%0} создано
-command.txt.edit=/txt edit
-command.txt.edit.success=Плавающий-Текст: отредактировано {%1} из {%0}
-command.txt.move=/txt m(ove)
-command.txt.move.success=Плавающий-Текст: перенесен {%0} to {%1}
-command.txt.remove=/txt r(emove)
-command.txt.remove.success=Плавающий-Текст: Удален {%0}
+command.txt.description=Управление летающими текстами
+command.txt.usage=/txt <a(dd)|e(dit)|m(ove)|r(emove)|l(ist)>
```
Comment (fuyutsuki) [inline resources/language/ru_ru.ini:23]: /txt list command is removed in 4.0.0

**Review by fuyutsuki — CHANGES_REQUESTED** [review]

**Comment by iteplenky** [comment]
> Please refer to [en_us.ini](https://github.com/fuyutsuki/Texter/blob/release/v4.0.4/resources/language/en_us.ini) and correct the old translation.
> Optionally, it would be great if you could also translate the README.md and place it as `.github/readme/ru_ru.md`.

I took en_us as a basis, and then translated it
The old translation was done entirely with google translate and some of the translations were missing.

**Comment by fuyutsuki** [comment]
Even now, some translations are not compatible with 4.0. so please check.

**Comment by iteplenky** [comment]
Are you sure? I mean I can't find what exactly is missing

**Comment by fuyutsuki** [comment]
In the current `ru_ru.md`, there are translates that were used in 3.x but not in the 4.x (e.g. `error.config.limit.char`). Please translate them based on [the current `en_us.md`](https://github.com/fuyutsuki/Texter/blob/release/v4.0.5/resources/language/en_us.ini).

**Comment by fuyutsuki** [comment]
Ah... okay, I'm sorry. You're right.
Thank you for the PR!

**Review by fuyutsuki — APPROVED** [review]

**Pull request merged**